### PR TITLE
Fixes occasional result of -0

### PR DESCRIPTION
**Pull request by mosschief**
Made changes to the compute_tokens function that checks for negative 0s by comparing result minus 0 to epsilon and sets them to 0 if found.

Addresses issue # 1 clicked around, got result -0. -0 results occurred after performing a calculation that resulted in a negative number and then multiplying the answer by 0. i.e. (5-6 = -1 then -1*0 = -0).

### CODE REVIEW

File: src/Core/Evaluation.vala [hunk 1]
```diff
@@ -263,6 +263,8 @@ namespace PantheonCalculator.Core {
             try {
                 Operator op = get_operator (t_op);
                 var d = (double)(op.eval (double.parse (t2.content), double.parse (t1.content)));
+                if (fabs (d) - 0.0 < double.EPSILON ) {
+		        d = 0; }
```
Comment (donadigo) [inline src/Core/Evaluation.vala]: Can you insert this brace here on new line?

**Comment by jeremypw** [comment]
Also, since d is double, should it not be "d = 0.0" on line 267?

**Comment by mosschief** [comment]
Hi @jeremypw I believe that is because calculations using functions like that are performed in the process_tokens function rather than the compute_tokens function. Adding changes to take care of that issue now.

**Comment by jeremypw** [comment]
Hint:  Try putting the same check in "process_tokens ()"

**Comment by jeremypw** [comment]
Thanks - looks like we commented with the solution simultaneously!

**Comment by mosschief** [comment]
Alright, made the changes. Also just removed the braces entirely as that seemed to be more in line with the style of the rest of the code for single line if statements.

**Comment by jeremypw** [comment]
Elementary code style now requires the format that donadigo indicated - even if existing code does not always comply.  So any code you add or touch should have the code style updated.

**Comment by mosschief** [comment]
Ok, sorry about that. Readded the braces on new line

**Comment by Philip-Scott** [comment]
Instead of having the same code on two separate places, would it work if you placed on a more general location, maybe something like like 86 in the evaluate function? :)

**Comment by jeremypw** [comment]
I agree these two functions could use some DRY-ing, while they are being modified.

**Comment by danrabbit** [comment]
Thanks for your fix @mosschief !

**Pull request merged**